docs/resources/notebook-authoring-guide.md

```diff
@@ -0,0 +1,147 @@
+# Notebook Authoring Accessibility Guide
+
+Writing notebooks is an art, depending on what as an author you want to share to the world the way that the writing and publishing techniques can vary widely. There are some writing and publishing techniques that you can use to ensure your notebooks' content is more accessible.
+
+In this section, we will introduce some guidelines to make edits to your notebooks for accessibility. It is an excellent step you can take to make our communities more inclusive!
+
+All the information presented is based on the [draft authoring tips](https://github.com/Iota-School/notebooks-for-all/blob/main/resources/event-hackathon/accessibility-tips-for-jupyter-notebooks.md) and the [Notebook authoring accessibility checklist](https://github.com/Iota-School/notebooks-for-all/blob/main/resources/event-hackathon/notebook-authoring-checklist.md) from the Notebooks For All team.
+
+# Table of contents
+1. [Structure](#structure)
+2. [Text](#text)
+3. [Code](#code)
+4. [Media](#media)
+    1. [Images](#images)
+    2. [Visualizations](#visualizations)
+    3. [Videos](#videos)
+    4. [Audio and Sonifications](#audio)
+    5. [Interactive Widgets](#interactive)
+5. [Resources](#resources)
+6. [Acknowledgements](#acknowledgements)
+
+## Structure <a name="structure"></a>
+
+Sighted users are able to perceive structure and relationship using different visual cues. We want to ensure that **information and relationships** are perceivable to all. In order to accomplish this goal, we as content authors need to provide access to the information in different modalities.
+
+We will discuss in this section some recommendations for the most common elements in the structure of your notebooks.
```

Comment:
Encouraging people to use content headings since they are currently the main means of structure in notebooks. 

```suggestion
At the time of writing, notebooks gain most of their structure from their content and content headings (like `<h1>`, `<h2>`, etc. or the Markdown equivalents `#`, `##`, etc.). Making sure to use headings and other markdown as intended is one of the most important things an author can do to make an approachable and navigable notebook. 
```